avro-fastserde/src/main/java/com/linkedin/avro/fastserde/SchemaAssistant.java

```diff
@@ -67,9 +71,10 @@ public static boolean isComplexType(Schema schema) {
   public static boolean isNamedType(Schema schema) {
     switch (schema.getType()) {
       case RECORD:
+        return true;
       case ENUM:
       case FIXED:
-        return true;
+        return Utils.isAvro14() ? false : true;
```

Comment:
avro 1.4 spec says fixed _are_ named types - https://avro.apache.org/docs/1.4.0/spec.html#Names
what does this condition affect ?

Reply:
if this is to avoid generating a getSchema() call (which 1.4 fixed classes dont have), i'd at least put a comment.

Reply:
You are right, I added a comment to mention the purpose.